src/RedemptionIdle.sol

```diff
@@ -43,11 +43,7 @@ contract RedemptionIdle is Redemption {
             / (usdPerUstbChainlinkRaw * USDC_PRECISION);
     }
 
-    /// @notice The ```redeem``` function allows users to redeem SUPERSTATE_TOKEN for USDC at the current oracle price
-    /// @dev Will revert if oracle data is stale or there is not enough USDC in the contract
-    /// @param to The receiver address for the redeemed USDC
-    /// @param superstateTokenInAmount The amount of SUPERSTATE_TOKEN to redeem
-    function redeem(address to, uint256 superstateTokenInAmount) external override {
+    function _redeem(address to, uint256 superstateTokenInAmount) internal {
```

Comment:
cant you put this on the abstract contract?

Reply:
https://github.com/superstateinc/onchain-redemptions/pull/37/commits/307eb726a416fd733c8e335acf435e4b5a0659de